src/kava.js

```diff
@@ -488,7 +488,7 @@ class Kava extends BasePlugin {
       maxSegmentDownloadTime: Math.max(seconds, this._model.maxSegmentDownloadTime),
       maxNetworkConnectionOverhead: lastFragResourceTiming
         ? Math.max(this._model.maxNetworkConnectionOverhead, lastFragResourceTiming.connectEnd - lastFragResourceTiming.domainLookupStart)
-        : 0
```

Comment:
@OrenMe , please notice that before if the fragment doesn't exist in the performance api then the entire calculation of the max would be turned to zero instead of leaving the max count same